src/tree.spec.ts

```diff
@@ -470,7 +470,7 @@ describe('makePictureTree', () => {
     };
     const expected = {
       img: {
-        src: 'https://cdn.filestackcontent.com/output=format:webp/quality=value:5/sepia=tone:70/seW1thvcR1aQBfOCF8bX',
+        src: 'https://cdn.filestackcontent.com/quality=value:5/sepia=tone:70/output=format:webp/seW1thvcR1aQBfOCF8bX',
```

Comment:
Unfortunately it will have a negative impact in this case scenario. If the input file was an image that doesn't support the quality setting, let's say a png image, the `quality` task wouldn't be applied, as the image would still be a png at this point. In the old example the image would be converted to webp before running the `quality` task, making it possible to apply the quality setting. Notice the difference in image quality in the below examples:
https://cdn.filestackcontent.com/quality=v:1/output=f:webp/LJNf24eDS1O7GsbpaEMs
https://cdn.filestackcontent.com/output=f:webp/quality=v:1/LJNf24eDS1O7GsbpaEMs

I'm not familiar with the `adaptive` package, but it looks to me that it would require a greater refactoring in order to make it possible to apply the `output` task as the last task in chain (which makes perfect sense) with being backwards compatible at the same time.

@pcholuj @sebastian-wec @promanski